### PR TITLE
Added assignment 3

### PR DESCRIPTION
**Pull request by PriyaaIyer**
Fixes #
- Added HTML Basics Remake folder
## Proposed Changes

-
-
-

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/35"></a>

### CODE REVIEW

**Comment by aravindvnair99** [comment]
@PriyaaIyer Next time fill up the pull request template properly.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Missing head tag. Refer this: [HTML Head - w3schools.com](https://www.w3schools.com/html/html_head.asp)

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Missing form identifier.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `required` attribute missing

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `required` attribute missing. And male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases. `label` needs to be used.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `required` attribute missing.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@PriyaaIyer make the necessary changes. Also, use tab 4 indentation:

- Press F1
- Convert indentation to tabs
- Press F1
- Indent using tabs
- Press F1
- Format Document

Henceforth, make changes in a separate branch. Here's why: [Squash merge and updating personal fork - Reddit](https://www.reddit.com/r/git/comments/8t97f8/squash_merge_and_updating_personal_fork/)

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Should be `required`

**Comment by PriyaaIyer** [comment]
Made the necessary changes as instructed by you.
Didn't quite understand the separate branch thing.
Also I am not able to create a new pull request for the changes that I have
made now

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `reg_form` or something of that sort would be a better name than `required`

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `required` attribute is missing.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `required` attribute is missing.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
> Made the necessary changes as instructed by you. Didn't quite understand the separate branch thing. Also I am not able to create a new pull request for the changes that I have made now

It's only from the next one onwards. Or if you wish to, you may close this pull request, add code to a different branch and send a pull request from that. Not necessary for this one though.

**Comment by PriyaaIyer** [comment]
Something is wrong with me....sorry for committing so many mistakes in such
a simple task.
I have added the required fields....

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: You have two forms starting here instead of one.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Two instead of one.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
Fix the other unresolved issues also.

**Comment by PriyaaIyer** [comment]
Indented using tabs is done
Formatting done using F1
Added all the conditions given in the document

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Missing autocomplete. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: `for` corresponds to `id`. Missing `id` attribute. Refer this: [HTML \<label> for Attribute - w3schools](https://www.w3schools.com/tags/att_label_for.asp)

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by PriyaaIyer** [comment]
Done

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: The form response would be `post` and not `get`. Refer this: [HTTP Request Methods - w3schools](https://www.w3schools.com/tags/ref_httpmethods.asp)

And also, don't use `on`. Use specific ones for each input. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by PriyaaIyer** [comment]
Used autocomplete 'on' option only for the necessary inputs. Haven't
specified otherwise.
Removed the one right next to form.
Form response changed.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Use the specific one for each input. Don't set to `on` as that's general.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Use the specific one for each input. Don't set to `on` as that's general.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Missing autocomplete.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Missing autocomplete.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by PriyaaIyer** [comment]
Added the specific autocomplete fields
Not sure with the 'given name' and 'family name' fields

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (aravindvnair99) [inline Priyaa/HTML Basics Remake/Assignment-3.html]:
```suggestion
        <input type="radio" name="gender" id="male" value="male" required>
```

Radio and checkboxes don't have autocomplete. `sex` autocomplete works for text input.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — DISMISSED** [review]
Seems fine to me from the code point of view. @VinayakTekade squash and merge (not merge commit) if you also approve (if answer meets the expected criteria).

@PriyaaIyer  Henceforth, make changes in a separate branch. Here's why: [Squash merge and updating personal fork - Reddit](https://www.reddit.com/r/git/comments/8t97f8/squash_merge_and_updating_personal_fork/) You can after this pull request is merged into master. Proceed as discussed on the call.

**Comment by PriyaaIyer** [comment]
Cool....I will do as told. Will go through the link you've sent across
after the call.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: The `email` should not have a minimum and maximum characters condition.

File: Priyaa/HTML Basics Remake/Assignment-3.html
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Use `label` instead of `b` to make your code more accessible.

**Review by VinayakTekade — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — APPROVED** [review]
@VinayakTekade squash and merge (not merge commit) if you also approve (if answer meets the expected criteria).

**Review by VinayakTekade — APPROVED** [review]

**Pull request merged**
